### ISSUE TITLE
Route '/photos' always returns content-type 'application/octet-stream'

### ISSUE THREAD

**Issue by gawin** [sinatra/sinatra#1002]
When writing a route named '/photos' it always returns a content-type 'application/octet-stream'.
```ruby
get '/photos' do
  'my photos'
end
```

Even if we I specify the content_type manually and/or use a templating engine, it still returns 'application/octet-stream'.
```ruby
get '/photos' do
  content_type 'text/html'
  erb :photos
end
```

Setting the content_type in a before filter also does not resolve the issue.
Note that other route names, for example /photo, /photos.html /picture, /pictures do not have this issue, it only occurs when naming the route '/photos'.
Example application: https://github.com/gawin/sinatra-content-type-bug

How can I force the content-type to 'text/html' while keeping the route named '/photos'?

**Comment by larryprice** [comment]
Running your app, I haven't been able to reproduce this locally: the page is opened with content_type 'text_html' for me. Screenshot:
![screenshot from 2015-05-05 14 54 54](https://cloud.githubusercontent.com/assets/1653573/7480487/d3a5286c-f336-11e4-8cda-b64182461d28.png)

Which version of Sinatra are you using? What web browser and OS are you using? Have you hard refreshed the page? Is there more context from your original application that might help?

**Comment by gawin** [comment]
Interesting, I tested it yesterday using Chrome 41.0.2272.104 (64-bit), Safari 7.1.5 and Firefox 37.0.2 on Mac OS X 10.9.5 and it kept giving 'application/octet-stream'. Tested with WEBrick 1.3.1 and Puma 2.11.2, in combination with Sinatra 1.4.6. Force cleared the cache on all browsers. Even trough in a proxy as an extra check.

Just did a reboot and that magically resolve the issue. Beats me, must be a glitch in the Matrix. Ticket can be closed. Thx for verifying @larryprice.
